jwst/assign_wcs/assign_wcs_step.py

```diff
@@ -59,7 +60,12 @@ def process(self, input):
                 reffile = self.get_reference_file(input_model, reftype)
                 reference_file_names[reftype] = reffile if reffile else ""
 
+            # Get the MSA metadata file if needed and add to reffiles
+            msa_metadata_file = input_model.meta.instrument.msa_metadata_file
+            if msa_metadata_file:
```

Comment:
Should it be checking for `None` here?

Reply:
You mean it should be
```if msa_metadata_file is not None:```
?

Reply:
May be. It's in the core schema and if it's not set (e.g. fixed slits) you get `None`, right?
Better try that before changing it.

Reply:
Since it is defined in the schema, if it is not populated, the line above just returns None.  `is not None` is probably the better syntax here.  I've changed it.